assets/js/common/SuseManagerSettingsDialog/SuseManagerSettingsModal.jsx

```diff
@@ -0,0 +1,151 @@
+            setUrl(value);
+            onClearErrors();
+          }}
+        />
+        <Label
+          className="col-span-2 self-start"
+          info="Only required for self-signed certificates"
+        >
+          CA Certificate
+        </Label>
+        {editingCertificate ? (
+          <Textarea
+            className="col-span-4"
+            value={certificate}
+            placeholder="Starts with -----BEGIN CERTIFICATE-----"
+            error={hasError('ca_cert', errors)}
+            onChange={({ target: { value } }) => {
+              setCertificate(value);
+              onClearErrors();
+            }}
+          />
+        ) : (
+          <div className="col-span-4 flex flex-row items-center justify-start p-5 border border-gray-200 rounded-md">
+            <EOS_LOCK_OUTLINED className="mr-3" size="25" />
+            <div>
+              <div>Certificate Uploaded</div>
+              <div className="text-xs">
+                {format(certUploadDate, "'Uploaded:' dd MMM y")}
+              </div>
+            </div>
+            <div className="flex flex-row grow justify-end">
+              <Button type="danger" onClick={() => setEditingCertificate(true)}>
+                Remove
+              </Button>
+            </div>
+          </div>
+        )}
+        <Label className="col-span-2" required>
+          Username
+        </Label>
+        <Input
+          className="col-span-4"
+          value={username}
+          placeholder="Enter a SUSE Manager username"
+          error={hasError('username', errors)}
+          onChange={({ target: { value } }) => {
+            setUsername(value);
+            onClearErrors();
+          }}
+        />
+        <Label className="col-span-2" required>
+          Password
+        </Label>
+        {editingPassword ? (
+          <Password
+            className="col-span-4"
+            value={password}
+            placeholder="Enter a SUSE Manager password"
+            error={hasError('password', errors)}
+            onChange={({ target: { value } }) => {
+              setPassword(value);
+              onClearErrors();
+            }}
+          />
+        ) : (
+          <div className="col-span-4 border border-gray-200 p-5 rounded-md">
+            <p className="inline align-sub leading-10">•••••</p>
+            <Button
+              className="float-right"
+              type="danger"
+              onClick={() => setEditingPassword(true)}
+            >
+              Remove
+            </Button>
+          </div>
+        )}
+
+        <p className="col-span-6">
+          <span className="text-red-500">*</span> Required Fields
+        </p>
+      </div>
+      <div className="flex flex-row w-80 space-x-2">
+        <Button
+          disabled={loading}
+          onClick={() => {
+            const payload = {
+              url,
+              username,
+              ...(editingCertificate && { certificate }),
```

Comment:
Let's just keep in mind that the api expects `ca_cert` in the payload: this ends up as `certificate`.

If we intend to have an extra transformation layer before sending to the API then ok.